### PR TITLE
Added container instructions for nodejs/expressjs integration

### PR DESCRIPTION
**Pull request by adnanrahic**
The docs didn't have an explanation of how to configure a Nodejs app running in container envs so it works with our agents.

### CODE REVIEW

File: docs/integration/express.js.md [hunk 1]
```diff
@@ -349,6 +349,272 @@ pm2 start app.js -i max
...
+FROM alpine
+WORKDIR /usr/src/app
+RUN apk add --no-cache --update nodejs
+COPY --from=build /usr/src/app/node_modules ./node_modules
+COPY . .
+EXPOSE <PORT>
+CMD ["node", "app.js"]
+```
+
+Change the `<PORT>` you're exposing and if needed, change `app.js` to the entrypoint of your application.
```
Comment (otisg) [inline docs/integration/express.js.md:410]: entrypoint => entry point

File: docs/integration/express.js.md [hunk 2]
```diff
@@ -349,6 +349,272 @@ pm2 start app.js -i max
...
+docker service create --name your-nodejs-app \
+  -p <PORT>:<PORT> \
+  --restart-condition any \
+  --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock \
+  -e MONITORING_TOKEN=<MONITORING_TOKEN> \
+  -e LOGS_TOKEN=<LOGS_TOKEN> \
+  <YOUR_IMAGE:TAG>
+```
+
+Alternatively, if you want to use a Docker Compose file with the `docker stack` command, add this to your `docker-compose.yml` file. First add your `<MONITORING_TOKEN>`, then change the `<PORT>` to the one you're exposing and `<YOUR_IMAGE:TAG>` to the name of the image you just built.
```
Comment (otisg) [inline docs/integration/express.js.md:466]: > add this

What is this referring to?  Stuff above or below?

File: docs/integration/express.js.md [hunk 3]
```diff
@@ -349,6 +349,272 @@ pm2 start app.js -i max
...
+        delay: 5s
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+    ports:
+      - "<PORT>:<PORT>"
+```
+
+
+### Kubernetes
```
Comment (otisg) [inline docs/integration/express.js.md:492]: Given the popularity of Kubernetes, would it make more sense to list this higher?

**Review by otisg — APPROVED** [review]
Minor stuff

File: docs/integration/express.js.md (hunk 2 above)
Comment (adnanrahic) [reply docs/integration/express.js.md]: Re-worded to say: `...snippet below...`

File: docs/integration/express.js.md (hunk 3 above)
Comment (adnanrahic) [reply docs/integration/express.js.md:492]: We have a nav on the right so people will see `Kubernetes`, they won't be able to miss it. Also, this ordering is the same as in the SC UI instructions. Main reason why I ordered it like this.

**Pull request merged**